pkgs/applications/video/lightworks/default.nix

```diff
@@ -0,0 +1,90 @@
+{ stdenv, fetchurl, dpkg, makeWrapper, patchelf, buildFHSUserEnv,
+  gtk3, gnome3, gdk_pixbuf, cairo, libjpeg_original, glib, gnome2, mesa_glu, nvidia_cg_toolkit, zlib, openssl, portaudio }:
```

Comment:
Please break overlong lines (>80 chars) like this:

```
{ stdenv ...
, gtk3 ...
}:
```

Note the placement of comma.